README.md

```diff
@@ -70,15 +76,22 @@ Fizz
 Buzz
 $ echo '{"foo": "bar", "fizz": {"buzz": ["1", "2", "Fizz", "4", "Buzz"]}}' | niet fizz.buzz -f squote
 '1' '2''Fizz' '4' 'Buzz'
-$ echo '{"foo": "bar", "fizz": {"buzz": ["1", "2", "Fizz", "4", "Buzz"]}}' | niet . -f yaml
+$ echo '{"foo": "bar", "fizz": {"buzz": ["1", "2", "fizz", "4", "buzz"]}}' | niet . -f yaml
```

Comment:
If you want ..... lol I do not understand why changing casse, but whatever

Reply:
me too lol I don't remember why...
I guess is due to the following output or something like that...

By the way it's not an issue

Reply:
no, not at all ;)